e2e/__tests__/wrongEnv.test.ts

```diff
@@ -24,7 +24,8 @@ describe('Wrong globals for environment', () => {
     assertFailuresAndSnapshot(['node', '-t=document']);
   });
 
-  onNodeVersions('<21', () => {
+  // Node.js 18 is the last LTS version, which is missing the global 'navigator'
+  onNodeVersions('<=18', () => {
```

Comment:
This test will be obsolete after dropping Node.js 18. So I though, perhaps keeping `18` here makes it easier to notice that?